.github/workflow-samples/kotlin-dsl/build.gradle.kts

```
@@ -19,7 +19,7 @@ tasks.test {
 
 tasks.named("test").configure {
     // Echo an output value so we can detect configuration-cache usage
-    println("::set-output name=task_configured::yes")
+    File(System.getenv("GITHUB_OUTPUT")).writeText("task_configured=yes\n")
```

This should probably be `appendText` to avoid overwriting previous output.
```suggestion
    File(System.getenv("GITHUB_OUTPUT")).appendText("task_configured=yes\n")
```

---

Thanks, but I'm having to replace this mechanism with a marker file. 
Since the value of `GITHUB_OUTPUT` changes on each run,  reading the environment variable itself causes the configuration-cache entry to be invalidated. This kind of defeats the purpose of this code, which is to verify that configuration caching was used!